mysql/changelog.d/16904.added

```diff
@@ -0,0 +1 @@
+DBMON-2051: Adding a deadlock metric to Maria DB
```

Comment:
The deadlock metric is not just for MariaDB right? It also applies to MySQL?

Reply:
true, will change. Although , seems counter is always 0 in MariaDB and older MySQL version. When I execute the same set on instructions manually from 2 connections I get the same error regarding the deadlock on all versions and types of DB. However the deadlock count is updated only on MySQL 8.0.

Reply:
if the deadlock count is only updated for MySQL 8.0, let's point this out in the changelog as well as in metadata.csv